jupyterhub/tests/test_api.py

```diff
@@ -159,6 +159,7 @@ def fill_user(model):
     model.setdefault('created', TIMESTAMP)
     model.setdefault('last_activity', TIMESTAMP)
     model.setdefault('servers', {})
+    model.setdefault('auth_state', '')
```

Comment:
I think this probably shouldn't change? If anything, it should be None instead of a str.